tests/integration/gs/test_basic.py

```diff
@@ -441,7 +507,9 @@ def test_lifecycle_config_bucket(self):
             LIFECYCLE_CONDITIONS_FOR_SET_STORAGE_CLASS_RULE)
         bucket.configure_lifecycle(lifecycle_config)
         xml = bucket.get_lifecycle_config().to_xml()
-        self.assertEqual(xml, LIFECYCLE_DOC)
+        self.assertTrue(
+            XmlCompare().xml_compare(ET.fromstring(xml),
```

Comment:
It seems unnecessary to need to instantiate an `XmlCompare` object to do this.  I'd suggest either making the methods in XmlCompare `@staticmethod`s, or moving them to the their own module and making them module-level methods (probably cleaner, since that would allow re-using those functions in other modules).